tests/command.addons.test.js

```diff
@@ -16,9 +16,8 @@ if (process.env.IS_FORK !== 'true') {
   })
 
   test.serial('netlify addons:list', async t => {
-    const regex = /No addons currently installed/
     const cliResponse = await callCli(['addons:list'], t.context.execOptions)
-    t.is(regex.test(cliResponse), true)
+    t.is(cliResponse.includes('No addons currently installed'), true)
```

Comment:
```suggestion
    t.true(cliResponse.includes('No addons currently installed'))
```

Would this be clearer? (also below)